articles/openshift/oshift-how-build-app-private-repo.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to build an OpenShift application from a private Github repo | UKCloud Ltd
```

Comment:
Github should be GitHub (change this throughout whole document).
Spell out repository in full.